backend/Testing/Fixtures/IntegrationFixture.cs

```diff
@@ -52,12 +60,29 @@ public void InitLocalFlexProjectWithRepo(ProjectPath projectPath)
         FileUtils.CopyFilesRecursively(_templateRepo, projectDir);
         File.Move(Path.Join(projectPath.Dir, "kevin-test-01.fwdata"), projectPath.FwDataFile);
         Directory.EnumerateFiles(projectPath.Dir).ShouldContain(projectPath.FwDataFile);
+    }
+
+    public async Task FinishLexboxProjectResetWithTemplateRepo(string projectCode)
+    {
+        await FinishLexboxProjectResetWithRepo(projectCode, _templateRepoZip);
+    }
 
-        // hack around the fact that our send and receive won't create a repo from scratch.
+    public async Task FinishLexboxProjectResetWithRepo(string projectCode, FileInfo repo)
+    {
+        var client = new TusClient();
+        client.AdditionalHeaders.Add("Cookie", $".LexBoxAuth={AdminJwt}");
+        var fileUrl = await client.CreateAsync($"{AdminApiTester.BaseUrl}/api/project/upload-zip/{projectCode}", repo.Length, [("filetype", "application/zip")]);
+        var responses = await client.UploadAsync(fileUrl, repo, chunkSize: 20);
+        responses.Select(r => r.StatusCode.ToString())
+            .ForEach((codeName) => codeName.ShouldBe(nameof(HttpStatusCode.NoContent)));
+    }
+
+    private static void InitRepoInDirectory(string projectDir)
```

Comment:
Because InitLocalFlexProjectWithRepo is using the template zip that already has the `.hg` folder added to it, calling InitRepoInDirectory would be unnecessary work (and the `hg init` step would fail because a `.hg` folder already exists).